skbuild/resources/cmake/FindNumPy.cmake

```diff
@@ -10,7 +10,10 @@
 #   The include directories needed to use NumpPy.
 # ``NumPy_VERSION``
 #   The version of NumPy found.
-#
+# ``NumPy_CONV_TEMPLATE``
+#   The command-line arguments required to execute the conv-template script
+# ``NumPy_FROM_TEMPLATE``
+#   The command-line arguments required to execute the from-template script
```

Comment:
I suggest to name the variables `NumPy_CONV_TEMPLATE_EXECUTABLE` and  `NumPy_FROM_TEMPLATE_EXECUTABLE`.

Since the variable is set to the path to the executable, should the description be updated to something like this:

```
Path to the conv-template script
```

Reply:
Well it's not necessarily the executable. It's supposed to be one or two arguments that you add to the beginning of the command line.

Reply:
Since `conv-template` and `from-template` are python scripts, using `find_program` will not work.

Instead I suggest the following approach:

```
if(NOT DEFINED NumPy_CONV_TEMPLATE_SCRIPT)
  execute_process(COMMAND "${PYTHON_EXECUTABLE}"
       -c "from numpy.distutils import conv_template; print(conv_template.__file__)"
    OUTPUT_VARIABLE _numpy_conv_template_file
    OUTPUT_STRIP_TRAILING_WHITESPACE
    ERROR_QUIET
  )
  set(NumPy_CONV_TEMPLATE_SCRIPT "${_numpy_conv_template_file}" CACHE FILEPATH "path to conv-template script")
endif()

...
```

Reply:
These scripts will be present in future releases of NumPy though (see [here](https://github.com/numpy/numpy/blob/5c5a215fa1101479ae9b8d127be32679c9f3f105/setup.py#L366-L367)). The `execute_process` is for backward compatibility only.